CHANGES.md

```diff
@@ -4,6 +4,10 @@
 
 #### @cesium/engine
 
+##### Breaking Changes :mega:
+
+- Changed behavior of `DataSourceDisplay.ready` to always stay `true` once it is initially set to `true`.
+
 ##### Fixes :wrench:
 
 - Fixed error when resetting `Cesium3DTileset.modelMatrix` to its initial value. [#12409](https://github.com/CesiumGS/cesium/pull/12409)
```

Comment:
Don't forget to add a link to the fixed issue as well. 😄

Reply:
I was waiting to see the PR number, updated :smile:

Reply:
Sorry, too soon 🏃‍♀️

Reply:
@jjspace I don't see a link to the fixed issue (https://github.com/CesiumGS/cesium/issues/12303). Can you please add it?